R/skip.R

```diff
@@ -44,13 +44,19 @@ skip_if_not <- function(condition, message = deparse(substitute(condition))) {
 
 #' @export
 #' @param pkg Name of package to check for
+#' @param minimum_version Minimum required version for the package
 #' @rdname skip
-skip_if_not_installed <- function(pkg) {
-  if (requireNamespace(pkg, quietly = TRUE)) {
-    return(invisible(TRUE))
+skip_if_not_installed <- function(pkg, minimum_version = NULL) {
+  tryCatch(
```

Comment:
I don't like the existing check for its side effect, it causes the package to be loaded. Furthermore, requireNamespace() doesn't check the version once the package is loaded. I could move the skip() call outside the tryCatch() if that helps